javaparser-core-serialization/src/main/java/com/github/javaparser/serialization/JavaParserJsonDeserializer.java

```diff
@@ -94,10 +115,10 @@ private Node deserializeObject(JsonObject nodeJson) {
             }
 
             Node node = nodeMetaModel.construct(parameters);
-            // Note: comment is a property meta model, but it is not listed as constructor parameter and not attached to node
-            // @see BaseNodeMetaModel.getConstructorParameters
-            if (parameters.containsKey("comment")) {
-                node.setComment((Comment)parameters.get("comment"));
+            // comment is in the property meta model, but not required as constructor parameter.
```

Comment:
nitpick: capital letter at the start of the comment
```suggestion
            // Comment is in the property meta model, but not required as constructor parameter.
```